SpringContainerMS/src/main/resources/application.properties

```diff
@@ -28,3 +28,10 @@ kcsolution.allocated.orders=allocated-orders
 kcsolution.containers=containers
 #Kafka topic name for ORDERS topics
 kcsolution.orders=orders
+#BPM service url
+kcsolution.bpm_anomaly=${BPM_ANOMALY}
```

Comment:
Are there ways to provide default values here?

Reply:
It seems so. The tricky part here was that we also need to set defaults in the docker compose file. Otherwise, the SpringBoot application will take the empty env variables coming from docker compose. Will add those in a different PR for the main refarch-kc project.